GameData/002_CommunityPartsTitles/Localization/CPT_ReStockPlus_loc.cfg

```diff
@@ -48,42 +48,42 @@ Localization
+		#LOC_RestockPlus_restock-fuel-tank-sphere-1875-1_title     = FS-18-220 Hemispherical Fuel Tank (RS+)                      // FL-TX220-R Hemispherical Liquid Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-sphere-25-1_title       = FS-25-400 Rockomax Hemispherical Fuel Tank (RS+)             // Rockomax X-200-R Hemispherical Liquid Fuel Tank
+
+		#LOC_RestockPlus_restock-fuel-tank-rcs-radial-tiny-1_title = OL-R-008 "Stratus V1" Miniature Monopropellant Tank (RS+)    // Stratus-V Miniature Monopropellant Tank
+		#LOC_RestockPlus_restock-fuel-tank-rcs-1875-1_title        = OL-18-400 RCS Fuel Tank (RS+)                        // FL-R4 RCS Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-rcs-375-1_title         = OL-37-k2 RCS Fuel Tank (RS+)                         // FL-S1 RCS Fuel Tank
+		#LOC_RestockPlus_restock-oretank-1875-1_title              = UH-18-800 Holding Tank (Medium) (RS+)                // Medium Holding Tank
+		#LOC_RestockPlus_restock-oretank-375-1_title               = UH-37-k04 Holding Tank (Grande) (RS+)               // Jumbo Holding Tank
+
+		#LOC_RestockPlus_restock-fuel-tank-1875-4_title            = FL-18-220 Fuel Tank (RS+)  // FL-X220 Liquid Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-1875-3_title            = FL-18-440 Fuel Tank (RS+)  // FL-X440 Liquid Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-1875-2_title            = FL-18-900 Fuel Tank (RS+)  // FL-X900 Liquid Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-1875-soyuz-1_title      = FV-18-k1 "Snazzy" Fuel Tank (RS+)// FL-S1200 Liquid Fuel Tank
+		#LOC_RestockPlus_restock-fuel-tank-1875-1_title            = FL-18-k2 Fuel Tank (RS+)  // FL-X1800 Liquid Fuel Tank
+
+		#LOC_RestockPlus_restock-fuel-tank-adapter-1875-125-1_title  = FV-12-18-C-600 Adapter Fuel Tank (Long) (RS+)   // FL-XA600 Fuel Tank Adapter
+		#LOC_RestockPlus_restock-fuel-tank-adapter-1875-125-2_title  = FV-12-18-A-160 Adapter Fuel Tank (Short) (RS+)  // FL-XA160 Fuel Tank Adapter
+		#LOC_RestockPlus_restock-fuel-tank-adapter-1875-0625-1_title = FV-06-18-160 Adapter Fuel Tank (RS+)            // FL-XA160-S Fuel Tank Adapter
+		#LOC_RestockPlus_restock-fuel-tank-adapter-25-1875-1_title   = FV-18-25-k1 Rockomax Adapter Fuel Tank (RS+)   // FL-XA1200 Fuel Tank Adapter
+
+		#LOC_RestockPlus_restock-fuel-tank-sphere-375-1_title        = FL-37-900 Hemispherical Fuel Tank (RS+) // Kerbodyne S3-1800R Hemispherical Liquid Fuel Tank
+		#LOC_RestockPlus_restock-nosecone-375-1_title                = FL-37-k04 Nosecone (RS+)                // Kerbodyne S3-3600 Nosecone
+		#LOC_RestockPlus_restock-fuel-tank-375-4_title               = FL-37-k02 Fuel Tank (RS+)               // Kerbodyne S3-1800 Tank
+		#LOC_RestockPlus_restock-fuel-tank-adapter-375-5-1_title     = XX FL-37 S3-S4-054 Adapter Tank (RS+)         // Kerbodyne SAIV Liquid Fuel Tank Adapter
```

Comment:
XX?

Reply:
This was a part that I could never find in game... I've gone back and defined it based on the settings with the ReStock+ mod itself.